kuma/javascript/src/bcd-signal.jsx

```diff
@@ -406,14 +416,23 @@ window.activateBCDSignals = (slug: string, locale: string) => {
      */
     const createBriefExplanationControl = () => {
         const headerText = gettext('Can you provide a brief explanation?');
-        const descriptionText = gettext(
-            'Briefly outline the issue you are highlighting.'
-        );
+        const descriptionText =
+            gettext('Briefly outline the issue you are highlighting.') +
+            ' ' +
+            interpolate(
+                gettext('Minimum %(min)s and maximum %(max)s characters.'),
+                {
+                    min: MIN_BRIEF_EXPLANATION,
+                    max: MAX_BRIEF_EXPLANATION.toLocaleString()
+                }
+            );
 
         const textAreaControl = document.createElement('textarea');
         textAreaControl.className = 'control-input';
         textAreaControl.id = 'brief-explanation';
-        textAreaControl.maxLength = 1000;
+        textAreaControl.maxLength = MAX_BRIEF_EXPLANATION;
+        textAreaControl.minLength = MIN_BRIEF_EXPLANATION;
```

Comment:
Flow is barfing on this line. For some reason it's OK with `maxLength` but not OK with `minLength`. Grr!